src/main/java/seedu/address/logic/commands/FindLeaveCommand.java

```diff
@@ -1,6 +1,8 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_LEAVES;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
```

Comment:
just wish to clarify real quick because i have not dealt with many commands or any command involving predicates. is there no other way for us to revert a list back from the filtered version to the original address/ leavesbook version?

Reply:
Based on the current model & model manager, this seems like the easiest method to revert the filtered list to the full list. I will double-check on this. Do you think this violates anything (coding convention, LoD etc)?

Reply:
No, I was just clarifying because I was unfamiliar with the predicate API. LGTM will approve now thanks for clarifying